src/main/java/Deadline.java

```diff
@@ -1,14 +1,21 @@
 public class Deadline extends Task {
 
-    private String by;
+    private final String dueDate;
```

Comment:
since dueDate is a constant, would upper snake case be more suitable to adhere to coding standard?